Packs/FullHunt/pack_metadata.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "FullHunt",
+    "description": "Integration with FullHunt, the attack surface database of the internet.\nFullHunt enables companies to discover all of their attack surfaces, monitor them for exposure, and continuously scan them for the latest security vulnerabilities.",
+    "support": "community",
+    "currentVersion": "1.0.0",
+    "author": "@sam0x90",
+    "url": "",
+    "email": "",
+    "created": "2023-05-28T20:23:11Z",
+    "categories": [],
```

Comment:
Please choose the best describing category for your integration from the following list :
['Analytics & SIEM', 'Utilities', 'Messaging and Conferencing', 'Endpoint', 'Network Security', 'Vulnerability Management', 'Case Management', 'Forensics & Malware Analysis', 'IT Services', 'Data Enrichment & Threat Intelligence', 'Database', 'Deception & Breach Simulation', 'Email', 'Identity and Access Management', 'Cloud Services', 'Authentication & Identity Management']